sp800-63a/sec6_dc.md

```diff
@@ -20,9 +20,11 @@ This section applies when the secondary authenticator is directly tied to the au
 1. Before issuance, the CSP SHALL verify the original authenticator status. The CSP SHALL NOT issue a derived authenticator if status indicates any type of termination, disablement, revocation, or expiration.
 2. Before issuance, the CSP SHALL verify that the corresponding authenticator is possessed and controlled by the claimant.  
 3. The derived authenticator SHALL be valid only as long as the subscriber is authorized to hold the original authenticator.
-4. The CSP SHALL record the details of the original authenticator used as the basis for derived authenticator issuance. 
-5. The CSP SHOULD set the expiration of the derived authenticator to the expiration of the primary authenticator. There are instances where the derived authenticator need not be directly tied to the expiration of the primary authenticator as the derived authenticator can provide authentication services in its place, for example, while the expiring primary credential is being replaced.
-6. The derived authenticator SHOULD be issued at an AAL equivalent to  the IAL the primary authenticator is bound to.  The derived authenticator MAY be issued at an AAL higher than the IAL the primary authenticator is bound to. For example, if the primary IAL is 2, the authenticator that is issued could be AAL2 or 3. The IAL the derived authenticator is bound to SHALL remain at the IAL of the primary authenticator.
+4. The CSP SHALL record the details of the original authenticator used as the basis for derived authenticator issuance. **MG: what are the details?**
```

Comment:
authenticator(s) since there is likely to be more than one authenticator involved.